public/locales/en/translation.json

```
@@ -949,8 +953,19 @@
     "viewAll": "View All",
     "eventsAttended": "Events Attended",
     "contactInfoHeading": "Contact Information",
+    "mobilePhoneNumber": "Phone no.",
+    "workPhoneNumber": "Work Phone no.",
+    "homePhoneNumber": "Home Phone no.",
+    "addressLine1": "Address Line 1",
+    "addressLine2": "Address Line 2",
+    "postalCode": "Postal Code",
+    "deleteUser":"Delete User",
```

Biome (1.9.4)

[error] The key deleteUser was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)